Cargo.toml

```diff
@@ -1,12 +1,16 @@
 [package]
 name = "libtransport"
 version = "0.0.1"
-authors = ["Samuel Marks <@SamuelMarks>"]
+authors = [
+    "Maxim Zakharov <@Maxime2>",
+    "Samuel Marks <@SamuelMarks>",
+    "Vladimir Komendantskiy <komendantsky@gmail.com>",
+]
 edition = "2018"
 
 [dependencies]
 libcommon-rs = { git = "https://github.com/Fantom-foundation/libcommon-rs" }
 serde = "1.0.97"
 bincode = "1.0.1"
 os_pipe = "0.8.0"
-futures-preview = { version = "=0.3.0-alpha.17", features = ["async-await", "nightly"] }
+futures-preview = { version = "=0.3.0-alpha.18", features = ["async-await", "nightly"] }
```

Comment:
Version change for futures-preview should be changed in all repos altogether to avoid version conflicts, so it is better to do in separate PR and having a reason to do so due to use features unavailable in current version, which is seem is not the case for this PR.

Reply:
OK. This is not really important. We can keep the version.